Multisig/Data/Services/Utils/DelegateWarningCalculator.swift

```diff
@@ -76,7 +76,7 @@ class DelegateWarningCalculator {
     ///   - addressInfoIndex: name index of addresses in the decoded transaction data
     /// - Returns: true if transaction is untrusted, false otherwise.
     static func isUntrusted(multiSendTx: MultiSendTx, addressInfoIndex: AddressInfoIndex?) -> Bool {
-        let result = multiSendTx.operation == .delegate && addressInfoIndex?.values[multiSendTx.to]?.addressInfo == nil
+        let result = multiSendTx.operation == .delegate
```

Comment:
this would make any delegate operation untrusted. I think this is not correct

Reply:
the problem is that if the transaction has the flag untrusted, it should still be untrusted. 

I think the fix should be in a different code line

Reply:
it seems like the line 23 is not triggered?

Reply:
This is only called on MultiSendTxs tha have the DELEGATE operation. According to Ivan, they should be warned about.

Reply:
However, that means we need to remove all checks of the name in the calculator file and in the tests 🤦